### ISSUE TITLE
Possible to create the feedme on a UserScript?

### ISSUE THREAD

**Issue by XtremePlayzCODE** [dahquan/rattlesnake#29]
Like I just said, would it be possible to create the feedme example on a UserScript?

**Comment by dahquan** [comment]
The UserScript would have to connect locally to the feedme example

**Comment by XtremePlayzCODE** [comment]
But still, not running on your P.C., just on the browser?

(I thought this because the "src" folder has non-node syntax, so it would be possible, right?)

**Comment by dahquan** [comment]
It would have to run on PC and the syntax is ES6. It's the newer version of JavaScript, it's later compiled into ES5 syntax (Take a look at the dist folder).

**Comment by XtremePlayzCODE** [comment]
The DIST not SRC folder... lol

**Comment by XtremePlayzCODE** [comment]
My main question is can there be a feeder bot in a userscript?

**Comment by dahquan** [comment]
I don't think it can just be done in a browser alone cause it requires using proxies to connect multiple bots

**Comment by XtremePlayzCODE** [comment]
True, true. What about putting a proxy list in the userscript?

**Comment by dahquan** [comment]
I don't think browsers allow you to change proxies for a specific connection, but I could be wrong. I'd have to take a look at it

**Comment by XtremePlayzCODE** [comment]
Broswec.
